cleanup-passes/travis.ts

```diff
@@ -34,6 +34,8 @@ interface TravisConfig {
       sources?: string[]
     }
   };
+  dist?: string,
+  sudo?: string,
```

Comment:
Optional, you can list the allowed values here as string literals. Kinda useful for type checking, more useful for documentation.

sudo?: 'false', 'required',